relay-server/src/endpoints/healthcheck.rs

```diff
@@ -70,4 +70,10 @@ pub fn configure_app(app: ServiceApp) -> ServiceApp {
         r.name("internal-healthcheck-live");
         r.get().with(liveness_healthcheck);
     })
+    // live check is also used to check network connectivity by downstream relays.
+    // It must have the same url as Sentry (hence two urls doing the same thing)
+    .resource("/api/0/relays/live/", |r| {
+        r.name("external-healthcheck-live");
+        r.get().with(liveness_healthcheck);
```

Comment:
it seems that when fetching this URL we really only care about the network errors and abandon the http response entirely. Is there a point in doing anything other than returning a 200?

You say this endpoint needs to be the same in Sentry, will this endpoint be implemented in sentry as well then? Would probably be good to have similar behavior there.

Reply:
Yes you are right, the possibility was actually put forward by @jan-auer. We could simply not implement the endpoint at all it and would return an `HTTP 400` which is not a network error and that would work. 

It was decided that that would be too confusing for anybody trying to understand what it is happening so we decided to just return the same info as the existing live message.

On the second point you are also right, I added the endpoint  in sentry as well, ( it is the endpoint that you helped me with debugging the routing problem a few days ago).